config/locales/en.yml

```diff
@@ -366,6 +366,7 @@ en:
       gethelp_header: Support
       actions_header: Your claim
       read_guide: Guide
+      exit_guide: Back to form
```

Comment:
where is this used?.. same applies to Line 379

looks like they shouldn't be here

@Demwunz